research-hub-web/src/app/app.component.ts

```diff
@@ -103,6 +122,10 @@ export class AppComponent implements OnInit, OnDestroy, AfterViewInit {
     } else {
       return { researchActivityIds: [item.id] };
     }
+
```

Comment:
Unnecessary spaces that can be removed.

Reply:
Well spotted! 👁️ 

Thought I might as well refactor it while I'm at it:

**Old**:
```TypeScript
  const type = item['type'];

  if (type === OptionType.Category) {
    return { categoryId: item.id };
  } else {
    return { researchActivityIds: [item.id] };
  }
```

**New**:

```TypeScript
getSearchQueryParams(item: any) {
  return item['type'] === OptionType.Category ? { categoryId: item.id } : { researchActivityIds: [item.id] };
}
```